android/assets/jsons/translations/template.properties

```diff
@@ -1,3 +1,12 @@
+# Language settings
+
+# Equivalent of a space in your language
+# If your language doesn't use spaces, just add "" as a translation, otherwise " "
+" " = 
+
+# If the first word in a sentence starts with a capital, translate this as 'true', otherwise as 'false'
```

Comment:
How about StartWithCapitalLetter, the word "Capital" has a different meaning in-game. I was confused by this.